samza-core/src/main/scala/org/apache/samza/system/chooser/BootstrappingChooser.scala

```diff
@@ -19,19 +19,12 @@
 
 package org.apache.samza.system.chooser
 
-import org.apache.samza.system.SystemAdmin
-import org.apache.samza.system.SystemStream
-import org.apache.samza.system.SystemStreamMetadata
-import org.apache.samza.system.SystemStreamPartition
-import org.apache.samza.system.IncomingMessageEnvelope
 import org.apache.samza.SamzaException
-import org.apache.samza.util.Logging
-import org.apache.samza.metrics.MetricsHelper
-import org.apache.samza.metrics.MetricsRegistryMap
-import org.apache.samza.metrics.MetricsRegistry
+import org.apache.samza.metrics.{MetricsHelper, MetricsRegistry, MetricsRegistryMap}
```

Comment:
Minor: Prefer keeping these as explicit imports (not  `_`) and on separate lines.